Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/InductionMotors/BaseClasses/CurrentBlock.mo

```diff
@@ -0,0 +1,27 @@
+within Buildings.Electrical.AC.ThreePhasesBalanced.Loads.MotorDrive.InductionMotors.BaseClasses;
```

Comment:
For most of the models in this base class package, the comments for each instance are missing.